src/Database/DriverFeatureEnum.php

```diff
@@ -0,0 +1,55 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         5.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Database;
+
+enum DriverFeatureEnum: string
```

Comment:
Do we need the suffix here? Like entities in application code, enums like this will become part of our domain model and I think a name like `DriverFeature` is clearer and easier to operate. We can add suffixes when aliasing if there are overlaps.

Reply:
I'm used to no suffix in other languages, but I wasn't sure if users would be confused since they're not used to eums via `::NAME`.

Reply:
Personally I would prefer keeping the `Enum` suffix to make them easily identifiable. Especially when they can be spread across the codebase and different namespaces.

Reply:
> I would prefer keeping the Enum suffix to make them easily identifiable. 

That's fair. We use suffixes for traits and interfaces as well.